code/modules/client/preference/preferences.dm

```diff
@@ -2248,63 +2234,6 @@ GLOB.glassesblock
 		character.dna.default_blocks.Add(GLOB.rp_stutterblock)
 
 
-	//Oh gods, why am I doing this? Kek, let's do it, I guess. ;-;
-	if(disabilities & DISABILITY_FLAG_WINGDINGS && (CAN_WINGDINGS in character.dna.species.species_traits))
-		if(disabilities_cure & CURE_FLAG_WINGDINGS)	//Curable Disabilities go onto the DNA_SE list
-			character.dna.SetDNAState(GLOB.wingdingsblock, TRUE, DNA_SE, TRUE)
-		else										//Incurable Disabilities go onto the DNA_RP list
-			character.dna.SetDNAState(GLOB.wingdingsblock, TRUE, DNA_RP, TRUE)	//Y'all killing me, I'm going to need to revamp setupgame.dm now x3
-		character.dna.default_blocks.Add(GLOB.wingdingsblock)
-
-	if(disabilities & DISABILITY_FLAG_FAT)
```

Comment:
Heh heh heh :3
I have wanted to get rid of these if statements for months.
Heh heh heh.
Heh heh heh.
kekekekekekekekekekeke.
I'm really excited xD